dateparser_cli/utils.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+
+DEFAULT_CACHE_DIR = os.environ.get("DEFAULT_CACHE_DIR", '~/.cache')
+DEFAULT_DIR_NAME = os.environ.get("DEFAULT_DIR_NAME", 'date_parser_models')
+
+date_parser_model_home = os.path.expanduser(os.path.join(DEFAULT_CACHE_DIR, DEFAULT_DIR_NAME))
+
+if not os.path.isdir(date_parser_model_home):
+    os.mkdir(date_parser_model_home)
```

Comment:
It's best to not put side-effects on the model error - usually we don't want stuff to happen on import, but only when we explicitly run something.